src/Microsoft.AspNetCore.Razor.Tools/compiler/resources/Microsoft.AspNetCore.Razor.ToolsImports.targets

```diff
@@ -0,0 +1,10 @@
+<!--
+Copyright (c) .NET Foundation. All rights reserved.
+Licensed under the Apache License, Version 2.0.
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(NuGetPackageRoot)microsoft.aspnetcore.razor.tools\*\build\**\*.targets"
```

Comment:
`$(NuGetPackageRoot)microsoft.aspnetcore.razor.tools\*\build`
What happens when there are multiple versions of the NuGet package in the cache? 
Can you instead do something like 
`$(NuGetPackageRoot)microsoft.aspnetcore.razor.tools\{0}\build` and replace the `{0}` while writing?